cardano-testnet/test/cardano-testnet-golden/Cardano/Testnet/Test/Golden/Config.hs

```diff
@@ -38,19 +38,19 @@ createConfigStringAndPath base era =
 createGoldenFilePath :: AnyCardanoEra -> FilePath
 createGoldenFilePath = \case
   AnyCardanoEra ByronEra ->
-    "cardano-testnet/test/cardano-testnet-test/files/golden/byron_node_default_config.json"
+    "cardano-testnet/test/cardano-testnet-golden/files/golden/byron_node_default_config.json"
```

Comment:
I think the paths in `haskell.nix` need to be updated too: https://github.com/input-output-hk/cardano-node/blob/master/nix/haskell.nix#L117 and in lines 193-199

Reply:
Done!